plugins/mcreator-localization/help/ru_RU/item_extension/burn_time.md

```diff
@@ -8,4 +8,4 @@
 `#число тиков = #число переплавляемых предметов * 200`
 
 Кликните [здесь](https://mcreator.net/wiki/burn-time-fuels), чтобы узнать время сгорания некоторых "ванильных"
-предметов.
+предметов.
```

Comment:
Change LS back, please.

Reply:
It's already the right one. :/

Reply:
What happens if you remove all the text here (Ctrl + A) and copy the text (Ctrl + A) of the old tip?